### PR TITLE
Pull request with type [CREATE] for user [matbe] and object type [Report] - titled [History of Remeditation for specific Compliance Item]

### PR DESCRIPTION
**Pull request by matbe**
Automated pull request from the Endpoint Configuration Manager Community hub.

### CODE REVIEW

File: objects/Report/History of Remeditation for specific Compliance Item.rdl [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
...DataSetName>DataSetAdminID</DataSetName><ValueField>UserSIDs</ValueField></DataSetReference></DefaultValue><Hidden>true</Hidden></ReportParameter><ReportParameter Name="locale"><DataType>String</DataType><DefaultValue><Values><Value>=User!Language</Value></Values></DefaultValue><Hidden>true</Hidden></ReportParameter><ReportParameter Name="LCID"><DataType>Integer</DataType><DefaultValue><DataSetReference><DataSetName>LCID</DataSetName><ValueField>ID</ValueField></DataSetReference></DefaultValue><Prompt>LCID</Prompt><Hidden>true</Hidden></ReportParameter><ReportParameter Name="CI_CIID"><DataType>String</DataType><Prompt>Configuration Item</Prompt><ValidValues><DataSetReference><DataSetName>param_ListCIs</DataSetName><ValueField>CI_ID</ValueField><LabelField>CIDisplayName</LabelField></DataSetReference></ValidValues></ReportParameter></ReportParameters><CodeModules><CodeModule>SrsResources, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35</CodeModule></CodeModules><rd:ReportUnitType>Mm</rd:ReportUnitType><rd:ReportServerUrl>https://s002as81pri.staden.gotheborg.net/ReportServer</rd:ReportServerUrl><rd:ReportID>b78a5471-ab85-4c57-9dda-46c2e5fa02e1</rd:ReportID></Report>
```
[the remark this answers is not in the document]
Comment (matbe) [reply objects/Report/History of Remeditation for specific Compliance Item.rdl]: @kmyrup Do I need to do anything more on my side before you can delete so I can try and upload it again?

[the remark this answers is not in the document]
Comment (kmyrup) [reply objects/Report/History of Remeditation for specific Compliance Item.rdl]: Branch has been deleted.  You are free to start over now.

Thanks for your patience. :)

[the remark this answers is not in the document]
Comment (matbe) [reply objects/Report/History of Remeditation for specific Compliance Item.rdl]: @kmyrup 
I still can't reupload it. It just get stuck on loading:
image

If I try another report it continues to the next step right away, but when trying this specific one it gets stuck. (I waited for 10 min).

It did dissapeared from the "Your hub" page when you closed this request but still doesnt work, I have tried restarting the server without any luck.

[the remark this answers is not in the document]
Comment (matbe) [reply objects/Report/History of Remeditation for specific Compliance Item.rdl]: I have also tried the following:
* Deleting the report and upload with another name
* Changed the GUID for ReportId and DatasourceID

But neither helped, still gets stuck on loading... :(
Is there a logfile on my side I can troubleshoot or is the problem on your side maybe?

[the remark this answers is not in the document]
Comment (kmyrup) [reply objects/Report/History of Remeditation for specific Compliance Item.rdl]: Can you check your admin console logs, as well as adminservice log?

[the remark this answers is not in the document]
Comment (matbe) [reply objects/Report/History of Remeditation for specific Compliance Item.rdl]: Did not find any obvious errors in those logs unfortunately.

[the remark this answers is not in the document]
Comment (matbe) [reply objects/Report/History of Remeditation for specific Compliance Item.rdl]: It seems there is something in the report that prevents it from working in the wizard.
I created a copy of my original report and removed the incorrect references before upload to SSRS and that report can be submitted to github. Both reports work in my SSRS and looks identical from a user perspective.
I created a "frown" in ConfigMgr and if anyone wants the exports of the two rdl to compare I can send it there.
Thank you for your help.

**Pull request closed without merging**